OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -119,6 +151,22 @@
     </codeblock>
 </div>
 
+<hr/>
+
+<h3 class="wiki-nav-item pb-2" id="CTL-x100 Android Mode">
+    Wacom CTL-x100 Android Mode
```

Comment:
It is not clear to which users are targeted by this wiki entry. Probably change to something that expresses "Wacom CTL-x100 is not detected by OpenTabletDriver, and uses only a portion of the area"

Users coming here will not know that their tablet is in Android mode.